www/css/coolhole.css

```diff
@@ -400,16 +433,166 @@
 }
 
 /* CoolPoints User Styling */
-#coinWrapper {
-  display: flex;
+#cpPromptBtn {
   grid-gap: 3px;
   gap: 3px;
+  height: 36px;
+}
+#cpModal {
+  padding: 0;
+}
+#cpModalContent {
+  padding-top: 5px;
+}
+#cpModalBody {
+  padding: 0;
+}
+#cpModalHeader {
+  display: flex;
+  padding: 5px 18px;
+  align-items: center;
+  border-bottom: none;
+}
+#cpHeaderBrand {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  > button {
+    margin-left: auto;
+    color: #ffffff;
+    opacity: 100%;
+  }
+}
+#cpPrompt {
+  padding: 0;
+}
+#cpPromptHead {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 5px 10px;
+  border-bottom: 1px white solid;
+  > h1 {
+    font-size: 48px;
+    font-style: italic;
+    font-weight: 600;
+  }
+}
+#cpPromptBody {
+  padding: 10px;
+  border-bottom: 1px white solid;
+}
+#cpPromptTerms {
+  color: gray;
+  font-size: 12px;
+  padding-left: 5px;
+  box-shadow: #0b0e0f80 0 10px 5px;
+}
+#cpPromptLegal {
+  overflow-y: scroll;
+  padding-left: 5px;
+  height: 75px;
+  font-size: 12px;
+  color: gray;
+  background-color: transparent;
+}
+#cpTab {
+  border-top: 1px solid;
+  border-bottom: 1px solid;
+  > .nav-item > .nav-link {
+    border-radius: 0;
+    border-bottom: none;
+    border-top: none;
+  }
+}
+#cp-under-construction {
+  display: flex;
+  justify-content: space-evenly;
+}
+.cpActionHeader:not([id="cpActionsCardHeader-Earnings"]) {
+  border-top: #ffffff50 1px solid;
+}
```

Comment:
Is Earnings excluded because it's the first? Probably better to use [`:not:first-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child) so we're not reliant on earnings always being first

Reply:
Damn, I must've missed that on MDN. I can use that instead.